src/main/java/io/vertx/core/shareddata/package-info.java

```diff
@@ -41,16 +45,16 @@
  * {@link examples.SharedDataExamples#example1}
  * ----
  *
- * === Cluster-wide asynchronous maps
+ * === Asynchronous maps
```

Comment:
rather `Asynchronous shared maps`

shared being either between event loops or cluster wide, it's important to keep in mind that this is not local